TestIdentity Failure in Vault 1.9.0

Vault 1.11 introduced the following error in test_identity.py. Creating an issue to track the resolution of this failure.

```
FAILED tests/integration_tests/api/secrets_engines/test_identity.py::TestIdentity::test_create_or_update_group_3_create_success_with_group_type - AssertionError: None != []
```

--- a/tests/integration_tests/api/secrets_engines/test_identity.py
+++ b/tests/integration_tests/api/secrets_engines/test_identity.py
@@ -1061,7 +1061,11 @@ class TestIdentity(HvacIntegrationTestCase, TestCase):
                     second=member_group_ids,
                 )
                 expected_member_entity_ids = (
-                    member_entity_ids if member_entity_ids is not None else []
+                    member_entity_ids
+                    if member_entity_ids is not None
+                    else []
+                    if group_type == "external" and utils.vault_version_lt("1.11")
+                    else None
                 )
                 self.assertEqual(
                     first=read_group_response["data"]["member_entity_ids"],

